src/api/spec/models/event_subscription/find_for_event_spec.rb

```diff
@@ -121,9 +180,13 @@
               create(:event_subscription_comment_for_project, receiver_role: 'maintainer', user: nil, group: nil)
             end
 
-            # TODO: This does not seem like the correct logic for this class.
-            it 'does not include that user/group' do
-              expect(subject.map(&:subscriber)).not_to include(user)
```

Comment:
@hennevogel this is the behaviour i have changed because it does not seem correct when i asked about it in a previous PR nobody was able to tell me what the desired behaviour is. This is why I had to change the assertion in `test/models/event_test.rb:166`

Reply:
This was the PR in which i raised this concern: https://github.com/openSUSE/open-build-service/pull/3952